lib/View/ContentEdit/ContentEditSuccessView.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryForms\View\ContentEdit;
+
+use eZ\Publish\Core\MVC\Symfony\View\BaseView;
+use eZ\Publish\Core\MVC\Symfony\View\View;
+use Symfony\Component\Form\Form;
+use Symfony\Component\Form\FormInterface;
+
+class ContentEditSuccessView extends BaseView implements View/*, FormView */
+{
+    /**
+     * @var \Symfony\Component\Form\FormInterface
+     */
+    private $form;
```

Comment:
Well, this might be part of the `FormView` interface (I don't wanna add methods to views that aren't part of an interface).

We could go for getForm / getFormData, but why would we provide the data separately (setter) ? We could as well return what's in the form, as a proxy (I'm not 100% sure what the benefit would be though.

But I'm also gonna be in need of an interface for `ContentEditSuccessView::getFormView()`... we have a tiny name conflict with both us and the Form component implementing a `FormView` interface ;-)